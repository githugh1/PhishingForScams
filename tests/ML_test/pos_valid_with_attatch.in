{
  "email_subject": "Project Proposal",
  "email_body": "Attached is the project proposal document...",
  "sender": "project@email.com",
  "attachments": ["proposal.pdf"]
}
