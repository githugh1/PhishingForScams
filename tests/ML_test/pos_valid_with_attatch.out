{
  "is_spam": false,
  "confidence": 0.25,
  "message": "This email is not considered spam."
}
